Python/pylifecycle.c

```diff
@@ -76,7 +76,22 @@ extern void _Py_ReadyTypes(void);
 extern void _PyGILState_Init(PyInterpreterState *, PyThreadState *);
 extern void _PyGILState_Fini(void);
 
-_PyRuntimeState _PyRuntime = _PyRuntimeState_INIT;
+//_PyRuntimeState _PyRuntime = _PyRuntimeState_INIT;
+_PyRuntimeState _PyRuntime = {
+    .initialized = 0,
+    .core_initialized = 0,
+    // Py_DecodeLocale() relies on PyMem_RawMalloc() and PyMem_RawFree()
+    // and may be needed before before runtime initialization.  To allow
```

Comment:
Doubled word "before before" in the comment.

Reply:
fixed